config/locales/en.yml

```diff
@@ -444,7 +472,7 @@ en:
       not_interested: Not interested in work right now
       not_interested_tag: Not interested
       open: Open, but not actively looking
-      open_tag: Open to offers
+      open_tag: Open
```

Comment:
Please revert this change.

Reply:
reverted in f786e1cef45f31d23935218e28516a5d4c236a73